src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,175 @@
+package duke;
```

Comment:
I think the naming of the methods and method parameters could be made clearer, I did an example on line 66.